scripts/image-builder/configure.sh

```diff
@@ -28,6 +28,15 @@ if grep -qE "Red Hat Enterprise Linux.*Beta" /etc/redhat-release; then
    sudo sed -i "s,dist/rhel9/${VID},beta/rhel9/9,g" /usr/share/osbuild-composer/repositories/rhel-"${VID}".json
 fi
 
+# Following edits composer's configuration to enable RT repository.
+# Duplicate first repository (baseos), change its name, and replace 'baseos' with 'rt'.
+# kernel-rt is only available for x86_64.
+"${SCRIPTDIR}/../fetch_tools.sh" yq
+sudo mkdir -p /etc/osbuild-composer/repositories/
+sudo "${SCRIPTDIR}/../../_output/bin/yq" \
+    '.["x86_64"] += (.["x86_64"][0] | .name = "kernel-rt" | .baseurl |= sub("baseos", "rt"))' \
+    /usr/share/osbuild-composer/repositories/rhel-9.4.json | jq | sudo tee /etc/osbuild-composer/repositories/rhel-9.4.json
```

Comment:
Should we make it more generic by `9.MINOR_VERSION`?

Reply:
Also, tee produces output, which we may want to redirect to /dev/null

Reply:
Changed, thx